src/main/java/crazypants/enderio/conduit/item/filter/ItemFilter.java

```diff
@@ -93,43 +90,60 @@ public boolean doesItemPassFilter(NetworkedInventory inv, ItemStack item) {
     return doesItemPassFilter(item);
   }
 
+  @SuppressWarnings("SimplifiableConditionalExpression")
   public boolean doesItemPassFilter(ItemStack item) {
-    if(!isValid()) {
-      return true;
+    boolean allow = isValid();
```

Comment:
That warning is emitted by IntelliJ for a reason... I find `allow = isValid() && itemMatched(item)` to be a lot more intuitive, though I do agree with you in using ternary operator with `isBlacklist`, instead of doing stuff like `isBlacklist != allow`.

Reply:
The IntelliJ only doubts about ternary operator :)
![image](https://user-images.githubusercontent.com/25156941/132108553-c3ead731-eb33-4433-aea7-0e2889b88901.png)

But ok, i changed it.